components/DiscussionPostMetadata.js

```diff
@@ -256,6 +248,12 @@ const DiscussionPostMetadata = (props) => {
   return (
     <div className={css(styles.container, containerStyle && containerStyle)}>
       <ContentSupportModal />
+      <ShareModal
+        isOpen={shareModalIsOpen}
+        setIsOpen={setShareModalIsOpen}
+        title={"Share this discussion"}
+        url={window.location.href + "#comments"}
```

Comment:
This could throw an error if we do a refactor because `window` is not defined on the server.
Could you see if you can use `${process.env.HOST}/router.pathname#comments` instead? The `HOST` env var is available on all environments.

Reply:
Good catch, I updated it.